timemachine/fe/geometry.py

```diff
@@ -0,0 +1,67 @@
+# Utility functions to help assign and identify local geometry points
+
+from enum import Enum
+
+from rdkit import Chem
+
+
+class LocalGeometry(Enum):
+    G1_TERMINAL = 0  # R-X
+    G2_KINK = 1  # R-X-H
+    G2_LINEAR = 2  # R-X#N
+    G3_PLANAR = 3  # R-X(=O)O
+    G3_PYRAMIDAL = 4  # R-X(-H)H
+    G4_TETRAHEDRAL = 5  # R-X(-H)(-H)H
+
+
+def assign_atom_geometry(atom):
+    """
+    Heuristic using hybridization information to assign local description
+    of geometry.
+    """
+    nbrs = list(atom.GetNeighbors())
+    if len(nbrs) == 0:
+        assert 0, "Ion not supported"
+    elif len(nbrs) == 1:
+        return LocalGeometry.G1_TERMINAL
+    elif len(nbrs) == 2:
+        if atom.GetHybridization() == Chem.HybridizationType.SP3:
```

Comment:
Readability preference: move RDKit API syntax (`atom.GetHybridization()`) out of conditionals.

(General preference: introduce aliases where needed, so that lines with key logic are as close as possible to a direct statement of intent:
`if atom.GetHybridization() == Chem.HybridizationType.SP3`
-->
`if hybridization == SP3`
or similar.)

Reply:
will fix

Reply:
For the `HybridizationType`, unfortunately I can't just import `from rdkit.Chem.HybridizationType import SP3` since it's not a module but an enum Class